api-reference/beta/resources/VerifiedCustomDomainCertificatesMetadata.md

```diff
@@ -0,0 +1,45 @@
+# VerifiedCustomDomainCertificatesMetadata resource type
+
+Represents the certificate details associated with the application published through Azure Active Directory Application proxy.
+
+### Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|thumbprint|String|The thumbprint of the certificate.<BR>Read-only|
+|subjectName|String|The subject name of the certificate.<BR>Read-only|
+|issuerName|String|The name of the issuer of the certificate.<BR>Read-only|
+|issueDate|datetime|The date and time the certificate was issued.<BR>Read-only|
+|expiryDate|datetime|The date and time the certificate will expire.<BR>Read-only|
```

Comment:
Did this go through API review?  This is not consistent with our naming guidelines.  the last 2 should be issueDateTime and expiryDateTime.  Confirmed with Sriram that it's OK to release to /beta like this, but this needs to get fixed.